src/ex1/style.css

```diff
@@ -0,0 +1,163 @@
+* {
+    font-family: Alef, Arial, Helvetica, sans-serif;
+    font-size: 20px;
+    margin: 0;
```

Comment:
in general, it's usually a bad practice to use the wildcard selector *, as it applies to all elements, rather than inheriting the style from a parent element (it still could be html or body) which are much easier to override when needed.
You can read more on CSS precedence here - https://jenkov.com/tutorials/css/precedence.html

You also have this cool library that normalizes the base/default styles across different browsers - https://necolas.github.io/normalize.css/